ci-operator/step-registry/msi/addon/install/msi-addon-install-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: msi-addon-install
+  from: ocp-addons-operators-cli
```

Comment:
Since these steps for `addons` are available to any job using steps in the step registry, I feel the `from` image should exist already within the build clusters image registries where the job runs on. Right now, the only way these steps (`msi-addon-install`, `msi-addon-uninstall`) would work is if the `dockerfile_literal` is defined in the prowgen job config to tell if the image to build/push to the internal image registry.

This [link](https://docs.ci.openshift.org/docs/how-tos/external-images/#mirror-public-images) explains more about mirroring the public image into the build clusters.

Reply:
will update in the base pr